docs/reference/glossary.md

```diff
@@ -0,0 +1,200 @@
+---
+slug: '/reference/glossary'
+displayed_sidebar: referenceSidebar
+---
+
+# Glossary
+
+## Use case {#use-case}
+
+_Use case_ or _integration use case_ describes what the business needs from a specific integration. Technically the use case defines an interface between the application and the integration provider, schema for input and output (result) data, possible error conditions, and documentation. It is part of the [profile](#profile).
+
+For example, the [Send SMS Message](https://superface.ai/communication/send-sms@2.0) (`SendMessage`) expects recipient's phone number, sender's phone number, and the text message contents. It will return a message ID as the result.
+
+Use case is written with the [Comlink Profile language](#comlink), for example:
+
+```hcl
+"""
+Send SMS Message
+Send single text message
+"""
+usecase SendMessage unsafe {
+  input {
+    "Recipient of the message"
+    to string
+
+    "Sender of the message"
+    from string
+
+    "The text of the message"
+    text string
+  }
+
+  result {
+    "Identifier of Message"
+    messageId string
+  }
+}
+```
+
+## Profile {#profile}
+
+_Profile_ contains one or more [use cases](#use-case) along with the version and a general documentation for the group of use cases. It is identified by a name with an optional scope separated with a slash, for example: `communication/send-sms`. Optionally, the profile can be scoped to a specific version, for example: `communication/send-sms@2.0.1`. Profiles are versioned with [Semantic Versioning](https://semver.org/) to facilitate for breaking changes.
```

Comment:
```suggestion
A _profile_ contains one or more [use cases](#use-case), along with the version and general documentation for that group of use cases. It is identified by a name, with an optional scope separated by a slash, for example: `communication/send-sms`. Optionally, the profile can be scoped to a specific version, for example: `communication/send-sms@2.0.1`. Profiles are versioned with [Semantic Versioning](https://semver.org/) to facilitate for breaking changes.
```

Reply:
Also "facilitate for breaking changes" doesn't work (sounds kind of like semver *helps you make breaking changes*)

- *signal* breaking changes?
- *handle* breaking changes?

Reply:
Good point, how about "indicate"?

```suggestion
	A _profile_ contains one or more [use cases](#use-case), along with the version and general documentation for that group of use cases. It is identified by a name, with an optional scope separated by a slash, for example: `communication/send-sms`. Optionally, the profile can be scoped to a specific version, for example: `communication/send-sms@2.0.1`. Profiles are versioned with [Semantic Versioning](https://semver.org/) to indicate breaking changes.
```

Reply:
Indicate! That works :)